test/libsolidity/util/ContractABIUtils.cpp

```diff
@@ -200,6 +201,8 @@ bool ContractABIUtils::appendTypesFromName(
 )
 {
 	string type = _functionOutput["type"].asString();
+	auto fixedPointType = frontend::FixedPointType::fromString(type);
```

Comment:
The tokenizer has this functionality, otherwise `fromString` is only used in isoltest, so maybe it would be best to just add the regex here.

Reply:
Moved the code from `FixedPointType::fromString(..)`.